pytorch_lightning/callbacks/early_stopping.py

```diff
@@ -86,21 +94,34 @@ def __init__(self, monitor: str = 'early_stop_on', min_delta: float = 0.0, patie
         # It is set to False initially and overwritten, if eval results have been validated
         self.based_on_eval_results = False
 
-        if mode not in self.mode_dict and mode != 'auto':
+        self.__init_monitor_mode()
+
+        self.min_delta *= 1 if self.monitor_op == torch.gt else -1
```

Comment:
where is the `self. monitor_op` set?
I can not see it inn init and `self.__init_monitor_mode()` uses `self.mode`

Reply:
it's a property.
https://github.com/PyTorchLightning/pytorch-lightning/blob/9a5d40aff4bbf8a0d72eae7d8a96e65fdefda234/pytorch_lightning/callbacks/early_stopping.py#L122-L124

Reply:
my bad, thx :]